modules/ossm-about-smcp.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-create-smcp.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-smcp_{context}"]
+= About ServiceMeshControlPlane
+
+The control plane includes Istiod, Ingress and Egress Gateways, and other components, such as Kiali and Jaeger. The control plane must be deployed in a separate namespace than the {SMProductShortName} Operators and the data plane applications and services. You can deploy a basic installation of the `ServiceMeshControlPlane`(SMCP) from the {product-title} web console or the command line using the `oc` client tool.
+
+[NOTE]
+====
+This basic installation is configured based on the default OpenShift settings and is not designed for production use. Use this default installation to verify your installation, and then configure your `ServiceMeshControlPlane` for your environment.
+====
+
+[NOTE]
+====
+Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where you can create resources and as a result the default deployment does not work. See xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-install-rosa_ossm-create-smcp[Installing Service Mesh on Red Hat OpenShift Service on AWS] for additional requirements before deploying your SMCP in a ROSA environment.
```

Comment:
```suggestion
Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where you can create resources, and as a result, the default deployment does not work. See xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-install-rosa_ossm-create-smcp[Installing Service Mesh on Red Hat OpenShift Service on AWS] for additional requirements before deploying your SMCP in a ROSA environment.
```
- Should ROSA be using an attribute?
- Add a comma before the coordinating conjunction
- We can't have xrefs in modules, so I think you need to remove the xref. The links will break on the Customer Portal

Reply:
When I look in openshift-docs/_attributes I don't see anything for ROSA.

The other changes have been made. Looks like dev  team is reversing decision and trying to release today. This is not a stop ship item, and I'm calling this done.